### DISCUSSION TITLE
How can i access the Directus DOM?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15716] [Archive]
My form is organized with groups in accordion.
Each group is very cleverly mounted by Directus only when it is open.

I have some Interface extensions in which I'd like to access a group after it is mounted.

Using the Component API way, I tried to import `ref `and `onMounted`.
But it looks like it works only for the HTML elements inside the `<template>...</template>` of the Interface

Is there a way to access with JavaScript the other fields of the group that Directus mounted on the group opening inside the accordion?

**Comment by rijkvanzanten** [comment]
While you theoretically could hack your way around with `document.querySelector`, I'd highly recommend avoiding direct DOM manipulation like that in a SPA framework context like Vue

**Comment by daawaan4x** [comment]
Hi @Pachat, do you mean accessing the other values of the form? You can call `inject('values')` to return a `ComputedRef<any>` of the form values

**Reply by Pachat** [reply]
Thanks. Is this call read-only?

**Reply by daawaan4x** [reply]
@Pachat yes, its read-only. if you want to update a value, you'll have to register `'set-field-value'` in the emits options of the component then call `emit('set-field-value', { field: 'foo', value: 'bar' })`
